cmd/syncEnumerator.go

```diff
@@ -241,3 +272,37 @@ func quitIfInSync(transferJobInitiated, anyDestinationFileDeleted bool, cca *coo
 		}, common.EExitCode.Success())
 	}
 }
+
+type CFDModeFlags struct {
+	// Change detection flags.
+	TargetCompare bool
+	CtimeMtime    bool
+	Ctime         bool
+	ArchiveBit    bool
+
+	// MetaDataOnlySync decides if we copy entire file even when change file detection finds out that only file's metadata has changed.
+	MetaDataOnlySync bool
+}
+
+func processCFDModeFlag(raw *rawSyncCmdArgs) CFDModeFlags {
```

Comment:
Tomar not added the check for if more than one cfdMode flag is set , as i have plan to change it enum/strings.